dispatch/api/serializers.py

```diff
@@ -522,6 +524,9 @@ class Meta:
             'section_id',
             'published_at',
             'is_published',
+            'is_breaking',
+            'breaking_timeout',
+            'currently_breaking',
```

Comment:
I'm thinking that since I added the currently_breaking field to the serializer that the 'is_breaking' field may not even be needed anymore